spec/introduction.md

```diff
@@ -0,0 +1,444 @@
+
+let db = schema.GetDataContext()
+```
+
+The type provider connects to the database automatically and uses this for IntelliSense and type information.
+
+Query expressions (added in F# 3.0) add the established power of query-based programming against SQL, Open Data Protocol (OData), and other structured or relational data sources. Query expressions provide support for language-Integrated Query (LINQ) in F#, and several query operators enable you to construct more complex queries. For example, we can create a query to filter the customers in the data source:
+
+```fsharp
+let countOfCustomers =
+    query {
+        for customer in db.Customers do
+            where (customer.LastName.StartsWith("N"))
+            select (customer.FirstName, customer.LastName)
+        }
+```
+
+Now it is easier than ever to access many important data sources—including enterprise, web, and cloud—by using a set of built-in type providers for SQL databases and web data protocols. Where necessary, you can create your own custom type providers or reference type providers that others have created. For example, assume your organization has a data service that provides a large and growing number of named data sets, each with its own stable data schema. You may choose to create a type provider that reads the schemas and presents the latest available data sets to the programmer in a strongly typed way.
+
+## Notational Conventions in This Specification
+
+This specification describes the F# language by using a mixture of informal and semiformal techniques. All examples in this specification use lightweight syntax, unless otherwise specified.
+
+Regular expressions are given in the usual notation, as shown in the table:
+
+| Notation          | Meaning                                  |
+| ----------------- | ---------------------------------------- |
+| regexp+           | One or more occurrences                  |
+| regexp*           | Zero or more occurrences                 |
+| regexp?           | Zero or one occurrences                  |
+| [ char - char ]   | Range of ASCII characters                |
+| [ ^ char - char ] | Any characters except those in the range |
+
+Unicode character classes are referred to by their abbreviation as used in CLI libraries for regular expressions—for example, `\Lu` refers to any uppercase letter. The following characters are referred to using the indicated notation:
+
+| Character | Name      | Notation                          |
+| --------- | --------- | --------------------------------- |
+| \b        | backspace | ASCII/UTF-8/UTF-16/UTF-32 code 08 |
+| \n        | newline   | ASCII/UTF-8/UTF-16/UTF-32 code 10 |
+| \r        | return    | ASCII/UTF-8/UTF-16/UTF-32 code 13 |
+| \t        | tab       | ASCII/UTF-8/UTF-16/UTF-32 code 09 |
+
+Strings of characters that are clearly not a regular expression are written verbatim. Therefore, the following string
+
+    abstract
```

Comment:
Not necessary now, but to recognize these blocks automatically, perhaps we can start using `abnf` sections (even though it isn't exactly abnf)?
`````suggestion
```abnf
abstract
```
`````